### PR TITLE
Lu 7489 delete survey

### PR DESCRIPTION
**Pull request by elthorne**

### CODE REVIEW

**Comment by elthorne** [comment]
Matthew, would you mind reviewing my Jinja2 code? I may have over-engineered it!

Paul, could you review my Python code please? :)

File: blaise_survey_manager/survey_config/functions.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from blaise_survey_manager.extensions import db
```
Comment (pds2208) [inline blaise_survey_manager/survey_config/functions.py:1]: Sadly the rest of the app is using SQLAlchemy's ORM so we should probably use it here as well

File: blaise_survey_manager/survey_config/views.py [hunk 2]
```diff
@@ -61,14 +63,26 @@ def list_surveys():
...
+    count_instruments = \
+        db.session.query(Survey, func.count(Instrument.survey_id)). \
+        outerjoin(Instrument, Instrument.survey_id == Survey.id). \
+        group_by(Survey.id)
+
+    instruments_per_survey = dict(count_instruments.all())
+
+    if len(instruments_per_survey) == 0 or all(instrument == 0 for instrument in instruments_per_survey.values()):
+        display_info_panel = False
+
```
Comment (elthorne) [inline blaise_survey_manager/survey_config/views.py:75]: @pds2208, by replacing the raw SQL with ORM I was able to refactor the rest of the code by removing the functions.py module and including everything in the list_surveys() in views.py.

I was even able to rename some variables and subsequently remove some comments! 

Overall improving code readability! 👍

**Comment by elthorne** [comment]
I broke the pagination! 😫

**Pull request closed without merging**